cypress.config.ts

```diff
@@ -0,0 +1,30 @@
+import { defineConfig } from 'cypress'
+
+export default defineConfig({
+    e2e: {
+        baseUrl: 'https://deriv.com',
+    },
+    env: {
+        email: 'test@example.com',
+        baseUrl: 'https://deriv.com',
```

Comment:
Let's do the test in staging, then we'll be able to test before code goes to production
```suggestion
        baseUrl: 'https://staging.deriv.com',
```